recipes/openssl/3.x.x/conanfile.py

```diff
@@ -1,4 +1,5 @@
 from conan.tools.files import rename
+from conan.tools.microsoft import is_msvc
```

Comment:
there is also a better logic using `msvc_runtime_flag` that you can pick up from 1.x.x branch